core/io/resource_format_binary.cpp

```diff
@@ -1122,7 +1112,7 @@ Error ResourceFormatLoaderBinary::rename_dependencies(const String &p_path, cons
 		memdelete(da);
 		//use the old approach
 
-		WARN_PRINT(("This file is old, so it can't refactor dependencies, opening and resaving: " + p_path).utf8().get_data());
+		WARN_PRINT(("This file is old, so it can't refactor dependencies, opening and resaving: " + p_path + ".").utf8().get_data());
```

Comment:
This could use `WARN_PRINTS` (the String version) to remove the need for the `.utf8().get_data()` C-string stuff.